packages/shared-influx-pieces/src/lib/utils/schedule.ts

```diff
@@ -0,0 +1,19 @@
+import { Time, TimerManager } from '@sapphire/time-utilities';
+import { container } from '@skyra/http-framework';
+import { getApproximateGuildCount } from './ApproximateGuilds.js';
+
+export function registerSchedule() {
+	return new Promise((resolve, reject) => {
+		try {
+			const analyticsSync = TimerManager.setInterval(async () => {
```

Comment:
Oh god no, not this completely unnecessary `TimerManager` bloat (sorry Vlad and Favna).

In fact, I discourage installing `@sapphire/time-utilities` in favour of its specific sub-packages because it pulls 4 libraries.